### ISSUE TITLE
KuiNameplates error originating from TRP3 alpha 1.7.0

### ISSUE THREAD

**Issue by Ellypse** [Total-RP/Total-RP-3#330]
```
Message: Interface\AddOns\Kui_Nameplates_Core\create.lua:664: attempt to compare number with nil
Time: Fri Jul 19 19:56:16 2019
Count: 2
Stack: Interface\AddOns\Kui_Nameplates_Core\create.lua:664: attempt to compare number with nil
[C]: ?
Interface\AddOns\Kui_Nameplates_Core\create.lua:664: in function <Interface\AddOns\Kui_Nameplates_Core\create.lua:631>
Interface\AddOns\Kui_Nameplates_Core\create.lua:691: in function <Interface\AddOns\Kui_Nameplates_Core\create.lua:682>
[C]: in function `UpdateNameText'
...ddOns\totalRP3\modules\NamePlates\Decorators\Kui.lua:297: in function `UpdateNamePlateName'
...ddOns\totalRP3\modules\NamePlates\Decorators\Kui.lua:443: in function `UpdateNamePlate'
...s\totalRP3\modules\NamePlates\DecoratorBaseMixin.lua:151: in function <...s\totalRP3\modules\NamePlates\DecoratorBaseMixin.lua:148>
(tail call): ?
...rface\AddOns\totalRP3\modules\NamePlates\Display.lua:342: in function `UpdateCustomizationState'
...ce\AddOns\totalRP3\modules\NamePlates\NamePlates.lua:108: in function `OnPlayerRegenEnabled'
...ce\AddOns\totalRP3\modules\NamePlates\NamePlates.lua:214: in function <...ce\AddOns\totalRP3\modules\NamePlates\NamePlates.lua:210>

Locals: 
```

**Comment by Ellypse** [comment]
I realized I forgot to give some context if you want to try to reproduce it. I was on the mecagone island doing quests, center south, where most of the pet battles are.

**Comment by Meorawr** [comment]
Going to close this as irrelevant with #510, but the speculative fix from 4fd5324 is also there anyway.
